jcstress-wrapper/src/main/kotlin/Main.kt

```diff
@@ -1,18 +1,27 @@
 import org.jetbrains.litmuskt.generateWrapperFile
 import org.jetbrains.litmuskt.generated.LitmusTestRegistry
+import kotlin.io.path.ExperimentalPathApi
 import kotlin.io.path.Path
+import kotlin.io.path.deleteRecursively
+import kotlin.io.path.div
 
+@OptIn(ExperimentalPathApi::class)
 fun main() {
     var successCnt = 0
     val allTests = LitmusTestRegistry.all()
+    val generatedSrc = jcstressDirectory / "generatedSrc"
+    try {
+        generatedSrc.deleteRecursively()
+    } catch (_: Exception) {
```

Comment:
Why the `catch` block is empty?
I think at the very least you might want to log the exception.

Reply:
This `delete()` is not really necessary, it's there only as an extra cleanup step, and all things should work fine even if this line fails completely. It was mainly useful during debugging

Reply:
Let's maybe then put `generatedSrc.deleteRecursively()` into a `finally` block?

Reply:
Sorry, I misinterpreted this piece of code at first. To avoid empty catch block, please use [`runCatching`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/run-catching.html) from stdlib.